src/main/scala/com/advancedtelematic/director/http/MultiTargetUpdatesResource.scala

```diff
@@ -40,11 +44,27 @@ class MultiTargetUpdatesResource(extractNamespace: Directive1[Namespace])(implic
 
   val route = extractNamespace { ns =>
     pathPrefix("multi_target_updates") {
-      (pathPrefix(UpdateId.Path) & get) { updateRequestId =>
-        getTargetInfo(updateRequestId, ns)
+      pathEnd {
+        post {
+          createMultiTargetUpdate(ns)
+        }
       } ~
-      (post & pathEnd) {
-        createMultiTargetUpdate(ns)
+      pathPrefix(UpdateId.Path) { updateId =>
+        pathEnd {
+          get {
+            getTargetInfo(updateId, ns)
+          }
+        } ~
+        path("apply") {
```

Comment:
I think we should avoid using verbs in a rest api. Not that our api is the best rest example but... 

This could be something like `PUT /devices/target/:update_id` ? Since we are actually setting the target of a set of devices